cmd/main.go

```diff
@@ -71,14 +75,14 @@ func main() {
 	if *provider.ConstructPEMChain {
 		klog.Infof("construct pem chain feature enabled")
 	}
-	if *provider.DriverWriteSecrets {
-		klog.Infof("secrets will be written to filesystem by the CSI driver")
+	if !*driverWriteSecrets {
+		klog.Infof("driver write secrets feature can't be disabled. The --driver-write-secret flag will be removed in future releases.")
```

Comment:
Should we do structured logging here?

Reply:
There are no key-values in this log statement, so this is also structure logged.